src/PanelTraits/Columns.php

```diff
@@ -94,33 +94,37 @@ public function addColumns($columns)
     /**
      * Moves the recently added column to 'before' the $target_col.
      *
-     * @param $target_col
+     * @param $target string|array the name or array of the target column
```

Comment:
The name of the target column or an array containing the 'name' key? (Is there better wording than that?)

Reply:
Tried to be consistent with other php docs but can change according to your suggestion